lisp/core.lisp

```diff
@@ -517,4 +517,14 @@ Example:
      (new-item old-item lst &rest mods)
      `(nsubstitute! ,new-item ,old-item (collect-copy ,lst) ,@mods))
 
+(defn *identity*
```

Comment:
Why the earmuffs?  We have never used them for functions before just other variables (otherwise then entire stdlib would be earmuffed).